front/app/containers/Admin/projects/project/nativeSurvey/FormResults/FormResultsQuestion/MappingQuestions/components/ExportGeoJSONButton.tsx

```diff
@@ -35,7 +36,7 @@ const ExportGeoJSONButton = ({ phaseId, customFieldId }: Props) => {
         `${API_PATH}/admin/phases/${phaseId}/custom_fields/${customFieldId}/as_geojson`,
         'application/geo+json'
       );
-      saveAs(blob, `${phaseTitle}_geojson.geojson`);
+      saveAs(blob, `${phaseTitle}_${moment().format('YYYY-MM-DD')}.geojson`);
```

Comment:
Is it not possible to read the filename from the BE response?

If not, then I probably can remove the code to construct the filename on the BE, and can just call it `export.geojson`, for example, every time it's exported, no?

Reply:
Ah I didn't realize the BE sends that back :thinking: I'll see if it's possible to use it! Thanks :raised_hands:

Reply:
Indeed - we can't get that from the Blob sent back by the BE. We have to set the filename on the FE :) So you can remove it from the BE I guess? :thinking: